code/modules/mob/living/simple_animal/eldritch_demons.dm

```diff
@@ -61,11 +59,48 @@
 	maxHealth = 50
 	health = 50
 	sight = SEE_MOBS|SEE_OBJS|SEE_TURFS
-	spells_to_add = list(/obj/effect/proc_holder/spell/targeted/ethereal_jaunt/shift/ash/long,/obj/effect/proc_holder/spell/targeted/telepathy/eldritch)
+	spells_to_add = list(/obj/effect/proc_holder/spell/targeted/ethereal_jaunt/shift/ash/long,/obj/effect/proc_holder/spell/pointed/manse_link)
```

Comment:
telepathy is still a useful spell for raw prophets to have for spooking purposes (stargazers get a form of telepathy too, for what it's worth

a shorter jaunt cooldown and/or an expanded range subtype of the blind spell would also be nice, just to make moving around maintenance easier (for the former) or to allow them to at least participate in combat (for the latter; even cult artificers get cult!magic missile to help support their team in combat, despite their dogshit direct damage output)

do demons (other than rust demons, armsy, and shapeshifted stalkers) have a way to receive healing? if not, letting raw prophets heal demons (including themselves) by clicking on them like their artificer counterparts can (for constructs) would be nice.

I know that this sounds like a lot, but raw prophets really do need a LOT of help if we're gonna make them a non-trap demon choice.